jsoncleaner.py

```diff
@@ -0,0 +1,56 @@
+import json
+import pandas as pd
+import numpy as np
+if __name__ == '__main__':
+
+df = pd.read_csv('merged.csv', index_col=None)
+
+
+def dolcomma(col):
+    return pd.to_numeric(col.map(lambda x: x.replace(',', '').replace('$', '')))
+
+
+# Cleaning up monetary columns and converting to integer
+df['Assessment Improvement Improved'] = dolcomma(df['Assessment Land'])
+df['Most Recent Sale Price Improved'] = dolcomma(df['Most Recent Sale Price'])
+df['Square Footage Improved'] = dolcomma(df['Square Footage'])
+df['Improvement Value Improved'] = dolcomma(df['Improvement Value'])
+df['Assessment Total Improved'] = dolcomma(df['Assessment Total'])
+df['Land Value Improved'] = dolcomma(df['Land Value'])
+df['Assessment Land Improved'] = dolcomma(df['Assessment Land'])
+df['Total Appraisal Value Improved'] = dolcomma(df['Total Appraisal Value'])
+
+# Cleaning up certain date columns
+df['Most Recent Sale Date'] = pd.to_datetime(df['Most Recent Sale Date'])
+df['Sale Date'] = pd.to_datetime(df['Sale Date'])
+
+# Converting 'Land Area' column to clean float
+df['Land Area Acres'] = df['Land Area'].map(lambda x: x.replace(
+    ' ', '').replace('Acres', '')).map(lambda x: float(x))
+
+# Neighborhood column is all numbers, but it's categorical.
+df.Neighborhood = df.Neighborhood.map(lambda x: str(x))
+
+# Need to convert this to numeric. Split-level = 1.5, 1.75 story= 1.5
```

Comment:
while this code works, this could probably be done more elegantly as:

```python
def map_stories(stories_string):
num_stories_dict = {'TWO STORY': 2, 'ONE STORY': 1, 'SPLIT-LEVEL': 1.5}
return num_stories_dict[stories_string
```

Also, why is 1.75 replaced with 1.5?